### PR TITLE
Write SQS unittests and Refactor receiveMsgFromSQS

### PR DESCRIPTION
**Pull request by JackPlowman**
Unit tests have been written for receiveMsgFromSQS and code around file-to-image-conversion has been updated to work with make unittests and other make targets

Changes 
- Unit tests written for receiveMsgFromSQS
- Renamed all variables/classes/files with misspelt receive
- Update make targets to include file-to-image-conversion
- Refactor receiveMsgFromSQS
- Bring file-to-image-conversion up to make code-check/code-format standards

### CODE REVIEW

File: application/file_to_image_conversion/tests/receiveMsgFromSQSTests.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        stubber.add_response("receive_message", receive_response, receive_expected_params)
+        stubber.add_response("delete_message", delete_response, delete_expected_params)
+        stubber.activate()
+        ReceiveMsgFromSQS = receiveMsgFromSQS()
+        ReceiveMsgFromSQS.client = sqs
+        sqs_response = ReceiveMsgFromSQS.consumeMsg(
+            queue_url="https://sqs.eu-west-2.amazonaws.com/000000000000/stub-address-queue.fifo"
+        )
+        stubber.assert_no_pending_responses()
+        assert str(sqs_response) in str(receive_response)
```
Comment (mabe13) [inline application/file_to_image_conversion/tests/receiveMsgFromSQSTests.py:132]: might be able to do this as just a compare of the dictionaries e.g. `sqs_response == receive_response["Messages"][0]`

File: application/file_to_image_conversion/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import sys, os
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
```
[the remark this answers is not in the document]
Comment (mabe13) [reply application/file_to_image_conversion/tests/__init__.py:3]: To not hold up the request of this PR being merged this issue will be resolved in a future Pull Request.

[the remark this answers is not in the document]
Comment (JackPlowman) [reply application/file_to_image_conversion/tests/__init__.py:3]: I will add a card to fix this issue so it doesn't get forgotten

[the remark this answers is not in the document]
Comment (JackPlowman) [reply application/file_to_image_conversion/tests/__init__.py:3]: This is card 88

**Pull request merged**